apis/extension/device_share.go

```diff
@@ -63,21 +63,19 @@ const (
 // DeviceAllocations would be injected into Pod as form of annotation during Pre-bind stage.
 /*
 {
-  "gpu": [
+  "rdma": [
```

Comment:
How about explain both gpu and rdma here and in the proposal? As a plain reader I might be confused what this annotation will look like when device type is gpu.

Reply:
Nice Suggesstion